pkg/processor/trigger/kafka/trigger.go

```diff
@@ -126,6 +126,10 @@ func (k *kafka) Start(checkpoint functionconfig.Checkpoint) error {
 
 	k.shutdownSignal = make(chan struct{}, 1)
 
+	if err = k.SignalWorkerContinue(); err != nil {
+		return errors.Wrap(err, "Failed to signal worker to continue")
+	}
```

Comment:
This should be inside the for loop below, so it will be called again after rebalance, before starting the consumption goroutines.